fakestorage/object.go

```diff
@@ -810,6 +810,7 @@ func (s *Server) downloadObject(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("X-Goog-Generation", strconv.FormatInt(obj.Generation, 10))
 	w.Header().Set("X-Goog-Hash", fmt.Sprintf("crc32c=%s,md5=%s", obj.Crc32c, obj.Md5Hash))
 	w.Header().Set("Last-Modified", obj.Updated.Format(http.TimeFormat))
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
Looking at the docs for GCS API, it seems that this should be configured in the bucket and honored by fake-gcs-server? https://cloud.google.com/storage/docs/configuring-cors#rest-apis

Reply:
Thanks for the pointer (and all your excellent work with this project)!

On quick look it seems the necessary work is:
1) add a handler for PATCH to ```/storage/v1/b/BUCKET_NAME?fields=cors``` that stores the configuration somewhere (do you have a pointer to existing code that shows the preferred way to store this type of config?)
2) load that config at this step and set the header.

Seems pretty straightforward, but being new to go and not having too much time to dive deeply, I'd appreciate a pointer to code with the preferred config set/get approach.

Reply:
Ugh, thinking a bit more about this, this one is tricky for the same versioning is tricky with the 

The filesystem backend currently doesn't support storing any bucket attributes, so that would be a requirement. I don't want to make this PR too large for you.

I opened #1025, we can merge this PR as is to get you unblocked.

I've merged some other changes though, so please rebase your PR so we can get it ready to ship.